cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -777,15 +777,6 @@
   </property>
 
   <property>
-    <name>data.tx.grace.period</name>
```

Comment:
why remove?

Reply:
So that it does not get documented. Is there a better way to do this?

Reply:
You can add it to cdap-docs/tools/cdap-default/cdap-default-exclusions.txt

Reply:
Done